flytepropeller/pkg/apis/flyteworkflow/v1alpha1/branch.go

```diff
@@ -56,6 +57,32 @@ type BranchNodeSpec struct {
 	ElseFail *core.Error `json:"elseFail,omitempty"`
 }
 
+// BranchNodeSpec contains at least one proto message directly, which means that we need to
+// implement DeepCopyInto since the generated proto messages do not implement that function.
+func (in *BranchNodeSpec) DeepCopyInto(out *BranchNodeSpec) {
```

Comment:
Discussed offline, let's keep the custom implementation to the proto types... and do so by wrapping them into a go struct